### DISCUSSION TITLE
[Solved] Resolution preset updates for Burn-in tool

### DISCUSSION THREAD

**Discussion by LeonCheung** [SubtitleEdit/subtitleedit#8335] [Ideas]
Continued with some discussion in #8210. After aggregating and referencing among many video converters, I think the current resolution preset list should be updated a bit, by removing some really rarely used ones and adding some more commonly used ones, and better including aspect ratio info as well if you agree with that:

Standard | Resolution | Aspect Ratio
-- | -- | --
4K DCI | 4096 × 2160 | 1.9:1
4K UHD | 3840 × 2160 | 16∶9
2K WQHD | 2560 × 1440 | 16∶9
2K DCI | 2048 × 1080 | 1.9:1
Full HD 1080p | 1920 × 1080 | 16∶9
HD 720p | 1280 × 720 | 16∶9
540p | 960 × 540 | 16∶9
SD PAL | 720 × 576 | 4:3
SD NTSC | 720 × 480 | 3:2
VGA | 640 × 480 | 4:3
360p | 640 × 360 | 16∶9
240p | 320 × 240 | 4:3

<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/zilan/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/zilan/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">

Vertical | Resolution | Aspect Ratio
-- | -- | --
TikTok / Y2B Shorts (A) | 1080 × 1920 | 9∶16
TikTok / Y2B Shorts (B) | 720 × 1280 | 9∶16
1/2 A | 540 × 960 | 9∶16
1/2 B | 360 × 540 | 9∶16
1/4 A | 270 × 480 | 9∶16
1/4 B | 180 × 270 | 9∶16

PLEASE NOTE: Some smaller dimensions are not really meant for modern and formal use, but are mainly used for saving the mobile phone data traffic while exchanging data with others.

One more idea would be:

SE should be able to detect and decide whether the video is horizontal or vertical (or square), and pop up a warning when the user carelessly sets a resolution in the **inverted aspect ratio**. Say, if the original video is vertical, but you set the resolution as horizontal, SE should warn that with a message.

I mention that because it seems one can easily mix up the resolution, say, 1920 x 1080 with 1080 x 1920. It would be terrible to notice that too late after batch generating so many wrong files already. 😂

**Comment by niksedk** [comment]
Could look like this:
![image](https://github.com/SubtitleEdit/subtitleedit/assets/1261331/d0470270-1395-4f5a-8084-c2ae86067341)

**Reply by LeonCheung** [reply]
Yes, just like that. :) The aspect ratio is useful for making sure that we always keep the same ratio when switching to another resolution rather than having to calculate manually.

**Reply by LeonCheung** [reply]
Oh just saw it's in beta. Thank you!
